.github/workflows/lint_markdown.yml

```diff
@@ -15,9 +15,9 @@ jobs:
       - name: Git checkout
         uses: actions/checkout@v3
       - name: Vale
-        uses: errata-ai/vale-action@13d79669d1c7cf33b26fc5607214ef529f071cdc
+        uses: errata-ai/vale-action@1edf87ba97278a700ba88ed9619172912ffaac25
         with:
-          vale_flags: "--glob=!{website/pages/blog/podcast-software-engineer-daily.md,*CHANGELOG.md,*/docs/tables/README.md,plugins/source/test/docs/tables/*.md,.github/styles/proselint/README.md,**/v1-migration.md,website/pages/docs/plugins/sources/*/tables.md,website/pages/docs/plugins/sources/*/policies.md,website/tables/**/*,website/pages/case-studies/*.mdx,cli/internal/docs/testdata/*.md}"
+          vale_flags: "--glob='!{website/pages/blog/podcast-software-engineer-daily.md,*CHANGELOG.md,*/docs/tables/README.md,plugins/source/test/docs/tables/*.md,.github/styles/proselint/README.md,**/v1-migration.md,website/pages/docs/plugins/sources/*/tables.md,website/pages/docs/plugins/sources/*/policies.md,website/tables/**/*,website/pages/case-studies/*.mdx,cli/internal/docs/testdata/*.md}'"
```

Comment:
Not sure since when but the glob pattern needs to be quoted I think https://github.com/errata-ai/vale-action/issues/37#issuecomment-1124012468